website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,317 @@
+# Operator Scope
+
+## Overview
+
+A namespace-scoped operator watches and manages resources in a single namespace, whereas a cluster-scoped operator 
```

Comment:
Can you break up lines in this doc, ex:
```suggestion
A namespace-scoped operator watches and manages resources in a single namespace,
whereas a cluster-scoped operator watches and manages resources cluster-wide.
```

Reply:
Hi @estroz, 

The IntelliJ shows a line with the expect line size/lint. All doc is respecting that so I cannot see a reason to do this break. See:

<img width="948" alt="Screenshot 2020-05-12 at 11 55 15" src="https://user-images.githubusercontent.com/7708031/81676320-9d3f9480-9447-11ea-8828-c66a8ba3d477.png">